### PR TITLE
chore(nodejs): update client ref docs link in metadata

### PR DESCRIPTION
**Pull request by fhinkel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ Apache Version 2.0
 
 See [LICENSE](https://github.com/googleapis/cloud-trace-nodejs/blob/master/LICENSE)
 
-[client-docs]: https://googleapis.dev/nodejs/trace/latest/
+[client-docs]: https://cloud.google.com/nodejs/docs/reference/trace-agent/latest
```
Comment (gkf) [inline README.md:282]: The new link is broken for me?

**Review by alexander-fenster — APPROVED** [review]

**Review by JustinBeckwith — CHANGES_REQUESTED** [review]
this 404s

**Comment by fhinkel** [comment]
Let me find out why the trace docs didn't get published

**Comment by fhinkel** [comment]
I kicked off the docs job, links is valid now.

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**